### ISSUE TITLE
[BUG] main thread panicks when running dol info command

### ISSUE THREAD

**Issue by emannotava** [encounter/decomp-toolkit#83]
### Game Name

Pokémon Rumble

### Game Version

USA v1

### Description

So after extracting Pokémon Rumble's WAD I tried to generate some info for its dol file (which is called 00000001.app but is the game's dol in reality)
And it almost immediately panicks and returns this error:
![image-190](https://github.com/user-attachments/assets/e8a2970a-f277-49a9-93f0-c0c2ae5564c7)

I've tried it using both the base game and the well-known Weekend Edition mod of the game which contains a few minor code changes.

However both yield the aforementioned result.

encounter themselves said it must be a bug over on the discord and told me I should report it as an issue on the github.

If I need to provide any files so you've got sample data to experiment with, do let me know.

**Comment by KooShnoo** [comment]
Pokémon Rumble's 00000001.app is compressed with a form of lzss, idk if its the same compression ppl call "nlzss" bc im pretty sure ther's multiple slightly different forms like "lz08" but idk who came up with all these names. I've heard the specific algorithm called "lz11" and "lz77ex". So, this statement:
> dol file [...] is called 00000001.app but is the game's dol in reality
is incorrect. 

tldr; that's not the dol, its a compressed archive of the dol.

maybe there's some way to validate that a dol file is valid, to give an error message instead of a panic?
